src/huggingface_hub/hf_api.py

```diff
@@ -853,7 +856,7 @@ def __init__(self, **kwargs):
             if safetensors
             else None
         )
-
+        self.security_repo_status = kwargs.pop("securityRepoStatus", None)
```

Comment:
> I'm not sure if it's a good idea to remove securityRepoStatus from the dict ([here](https://github.com/huggingface/huggingface_hub/blob/f005744f31b50a46b87ca4106f1d8cd3789211db/src/huggingface_hub/hf_api.py#L859)), this will implicitly "break" things again since users might be using this key already. what do you think?

I think it's fine to break it given how recent this change is

Reply:
yes, agree